includes/Plugin.php

```diff
@@ -197,6 +197,7 @@ function() use ( $options, $activation_flag ) {
 				( new Core\User_Surveys\REST_User_Surveys_Controller( $authentication ) )->register();
 				( new Core\Util\Migration_1_3_0( $this->context, $options, $user_options ) )->register();
 				( new Core\Util\Migration_1_8_1( $this->context, $options, $user_options, $authentication ) )->register();
+				( new Core\Modules\Module_Sharing_Settings( $options ) )->register();
```

Comment:
The `Module_Sharing_Settings` class `register()` method has not been called in the `Plugin.php` `init` hook as a part of #4475. Hence, adding it here because this PR changes requires it.
[Slack reference](https://10up.slack.com/archives/CBKKQEBR9/p1643127443230100).